src/tribol/mesh/MeshData.hpp

```diff
@@ -372,7 +393,7 @@ class MeshData {
     const MultiViewArrayView<RealT> m_response;
 
     /// Array view of 2D nodal normal data
-    const Array2DView<RealT> m_node_n;
+    Array2DView<RealT> m_node_n;
```

Comment:
Per my coupling scheme comment about moving nodal normal method information down to mesh data, this m_node_n might get swallowed up in that nodal normal object if it moves down here.